plugins/SpectrumAnalyzer/Analyzer.cpp

```diff
@@ -28,8 +28,13 @@
 #include "Analyzer.h"
 
 #include "embed.h"
+#include "lmms_basics.h"
 #include "plugin_export.h"
 
+#ifdef SA_DEBUG
```

Comment:
This must be moved up to match our include conventions.

Reply:
The conventions only say to group by type, not which type comes first. I left these on the bottom, because it looked visually better than breaking the flow of includes in the middle. But as you wish, I will use the more common order.